src/helpers.js

```diff
@@ -399,8 +399,8 @@ export function findHostNodeDevice(hostdev, nodeDevices) {
     case "usb": {
         const vendorId = hostdev.source.vendor.id;
         const productId = hostdev.source.product.id;
-        const device = parseInt(hostdev.source.device, 16).toString();
-        const bus = parseInt(hostdev.source.bus, 16).toString();
+        const device = hostdev.source.device;
+        const bus = hostdev.source.bus;
```

Comment:
These variables are not values which are shown in the UI (therefore not related to the bug). They merely are there so we can compare output of virsh dumpxml and virsh nodedev-dumpxml (i.e. match hostdev in VM's XML to a nodedev provided by libvirt)
Example of output of virsh nodedev-dumpxml:
```
<device>
  <name>usb_1_10</name>
  <path>/sys/devices/pci0000:00/0000:00:14.0/usb1/1-10</path>
  <devnode type='dev'>/dev/bus/usb/001/003</devnode>
  <parent>usb_usb1</parent>
  <driver>
    <name>usb</name>
  </driver>
  <capability type='usb_device'>
    <bus>1</bus>
    <device>3</device>
    <product id='0x0026'>AX201 Bluetooth</product>
    <vendor id='0x8087'>Intel Corp.</vendor>
  </capability>
</device>
```

Example of hostdev from virsh dumpxml:
```
    <hostdev mode='subsystem' type='usb' managed='yes'>
      <source>
        <vendor id='0x8087'/>
        <product id='0x0026'/>
        <address bus='1' device='3'/>
      </source>
      <address type='usb' bus='0' port='4'/>
    </hostdev>
```

Since both bus and device values are decimal numbers, we can just take them as strings and compare them.

Reply:
I see, thanks; resolving